setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
     #roman_datamodels>=0.12.2
     roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git@main
     stcal>=0.7.2
-    stpipe==0.3.1
+    stpipe>=0.4.1
```

Comment:
Now that stpipe was released we should update this to >=0.4.2 <1

Reply:
done in 918904e